0015-automatic-cluster-upgrades/0015-automatic-cluster-upgrades.md

```diff
@@ -0,0 +1,139 @@
+# RFC 0015 - Automatic Cluster Upgrades
+
+## User stories
+
+- As a user, and also as KaaS team provider, I would like to have the clusters running in the latest version automatically, and not worry about CVEs or EOL of the cluster components.
+
+- As a user, I would like to define maintenance windows to ensure clusters are not upgraded outside of these hours.
```

Comment:
```suggestion
- As a customer admin, I would like to define maintenance windows to ensure workload clusters are only upgraded within these hours.
```

user -> customer admin, to be more specific.

Changed into positive language instead of negative.